sproutify.js

```diff
@@ -86,7 +100,8 @@ const map = {
   candyCorn: candyCornImagePaths,
   eggplants: eggplantImagePaths,
   potatoes: potatoImagePaths,
-  tomatoes: tomatoImagePaths
+  tomatoes: tomatoImagePaths,
+  pumpkins: pumpkinImagePaths
```

Comment:
This one too, please put it above the `tomatoes` key so it's in alphabetical order 😊